tests/testthat/test-gather.R

```diff
@@ -88,24 +88,6 @@ test_that("overwrites existing vars", {
 
 # Column types ------------------------------------------------------------
 
-test_that("can gather all atomic vectors", {
```

Comment:
Why did you remove these tests?

Reply:
Because the one below is exactly the same, but does even more. It even has the same name "can gather all atomic vectors".

Reply:
Oh, got it!